### ISSUE TITLE
TODO: Add sync option for syncing data in db as well as file hierarchy

### ISSUE THREAD

**Issue by yunjae2** [yunjae2/tagit#4]
It is quite common to use the data directly, so add an option to maintain data in file hierarchy sync'ed as well as in db.

Example: 
```
$ tagit record -s /my/path/ myexp "a=1, b=2" -- ./run_exp.sh
```
```
$ tagit report -s /my/path/ myexp
```

**Comment by yunjae2** [comment]
What if user updates the file hierarchy?
tagit is not aware of the update currently.

**Comment by yunjae2** [comment]
This can be implemented as a part of the issue #15.
